services/ui-src/.env_example

```diff
@@ -11,9 +11,9 @@ COGNITO_USER_POOL_CLIENT_DOMAIN=main-login-21guh5mqnb6otpoli8o9irlj94.auth.us-ea
 COGNITO_USER_POOL_ID=us-east-1_z8B48Bul1
 LD_PROJECT_KEY=mdct-mfp
 LOCAL_LOGIN=true
-POST_SIGNOUT_REDIRECT=https://test.home.idm.cms.gov/app/UserHome
+POST_SIGNOUT_REDIRECT=http://localhost:8000
```

Comment:
Can we change this? I know its fine it when running local but is it set like `https://test.home.idm.cms.gov/app/UserHome` because its also used in the dev environment? @BearHanded cause I think you added this

Reply:
This should only matter for local dev since it's just being copied for local dev .env files. The dev, val, and prod variables come from AWS for those envs.

Reply:
@ajaitasaini Should be good to resolve this convo and merge. 👍